### PR TITLE
DRAFT: #23 Base webapp proposal creator

### PR DESCRIPTION
**Pull request by reubenstein**
Issue #23

### CODE REVIEW

**Comment by sktbrd** [comment]
Added this PR in Pod-tech Agenda in Discord and Discussions

File: package.json [hunk 1]
```diff
@@ -11,7 +14,7 @@
     "turbo": "^1.13.2"
   },
   "engines": {
-    "node": "18"
+    "node": ">=18"
```
Comment (bagelface) [inline package.json:17]: I would like to preface this review with saying i'm not a frontend dev, so I might ask some dumb questions. That being said, is there are reason this cannot be locked to "node 18" and needs to allow for ">= 18". Seems it would be best to be precise.

File: packages/gnars-contracts/README.md [hunk 2]
```diff
@@ -5,9 +5,12 @@ This project demonstrates a basic Hardhat use case. It comes with a sample contr
...
-REPORT_GAS=true yarn hardhat test
-yarn hardhat node
-yarn hardhat run scripts/proposeAuctionHouseUpgrade.ts
+pnpm hardhat help
+pnpm hardhat test
+REPORT_GAS=true pnpm hardhat test
+pnpm hardhat node
+pnpm hardhat run scripts/proposeAuctionHouseUpgrade.ts
+
+Here’s a draft script for our broadcast tomorrow. Feel free to make any changes you want!
```
Comment (bagelface) [inline packages/gnars-contracts/README.md:14]: Why is this line here?

File: packages/gnars-contracts/hardhat.config.ts [hunk 3]
```diff
@@ -55,7 +62,7 @@ const config: HardhatUserConfig = {
   },
   etherscan: {
     apiKey: {
-      mainnet: process.env.ETHERSCAN_API_KEY!,
```
Comment (bagelface) [inline packages/gnars-contracts/hardhat.config.ts:58]: probably fine to leave this line since the "mainnet" network is still defined above

File: packages/gnars-contracts/package.json [hunk 4]
```diff
@@ -25,13 +24,16 @@
...
+    "ethers": "^5.7.2",
     "hardhat": "^2.13.0",
     "hardhat-abi-exporter": "^2.10.1",
     "hardhat-gas-reporter": "^1.0.9",
     "solidity-coverage": "^0.8.2",
     "typechain": "^8.1.1"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.8.2"
+    "@openzeppelin/contracts": "^4.8.2",
```
Comment (bagelface) [inline packages/gnars-contracts/package.json:35]: this can probably be a dev dependency

File: packages/gnars-hd/contract/lib/solmate/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "solmate",
+  "name": "@gnars/solmate",
```
Comment (bagelface) [inline packages/gnars-hd/contract/lib/solmate/package.json:2]: why is this getting changed?
